### PR TITLE
Google.AdsEditor version 1.3.36.152

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99468)

### CODE REVIEW

File: manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.AdsEditor
+PackageVersion: 1.3.36.152
+PackageLocale: en-US
+Publisher: Google
+PackageName: Google Ad Editor
```
Comment (Trenly) [inline manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.locale.en-US.yaml:8]:
```suggestion
PackageName: Google Ads Editor
```

File: manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.AdsEditor
+PackageVersion: 1.3.36.152
```
Comment (Trenly) [inline manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 14.2.2.0
```
Writes 14.2.2.0 to the registry

File: manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Google.AdsEditor
+PackageVersion: 1.3.36.152
```
Comment (Trenly) [inline manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.yaml:5]:
```suggestion
PackageVersion: 14.2.2.0
```
Writes 14.2.2.0 to the registry

File: manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.AdsEditor
+PackageVersion: 1.3.36.152
```
Comment (Trenly) [inline manifests/g/Google/AdsEditor/1.3.36.152/Google.AdsEditor.installer.yaml:5]:
```suggestion
PackageVersion: 14.2.2.0
```
Writes 14.2.2.0 to the registry

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manifest validation error:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\g\Google\AdsEditor\1.3.36.152', expected 'manifests\g\Google\AdsEditor\14.2.2.0'

These have to match, but currently do not:
image

**Comment by upintheairsheep** [comment]
Fixed the version issue.

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installer did not complete:
image

**Comment by upintheairsheep** [comment]
The installer seems not to have a silent switch.

**Comment by upintheairsheep** [comment]
There is an issue to filter them, but until then, this is to leave

**Pull request closed without merging**